docs/wiki/Market-Making.md

```diff
@@ -0,0 +1,162 @@
+
+
+This page is part of the [Using Bison Wallet](Using-Bison-Wallet) guide, it assumes you have already 
+set up a Bison Wallet using the [Getting Started Guide](Getting-Started).
+
+---
+
+Bison Wallet offers integrated market-making bots that allow users to strategically place orders on both sides of the order book 
+and across supported centralized exchanges. This approach aims to capture potential profits through the spread and arbitrage opportunities, while also enhancing market stability. By providing liquidity and minimizing slippage, market makers help create 
+a healthier and more stable trading environment.
+
+
+> [!CAUTION]
+> Using market-making functionality involves risk to your capital. Market conditions can change rapidly, potentially leading to 
+losses from price fluctuations or market volatility. Ensure you understand these risks fully before participating in market making, 
+as there is no guarantee of profit, and you may incur losses.
+
+The market making functionality can be accessed from the header.
+
+<img src="./images/using-bison-wallet/header-mm.png" width="400" alt="">
+
+# Bot Management
+
+Upon opening the market making screen, you can create a new bot, edit an existing bot, configure API access for centralized exchanges, or review previous run logs.
+
+<img src="./images/using-bison-wallet/mm-start.png" width="800" alt="">
+
+When creating a new bot, you will be prompted to select the market you want it to trade in. If you have configured API access for supported centralized exchanges, an icon will highlight the markets that support arbitrage strategies.
+
+<img src="./images/using-bison-wallet/mm-selectmarket.png" width="400" alt="">
+
+After selecting the desired market, a prompt will appear, allowing you to choose one of the supported bots.
+
+<img src="./images/using-bison-wallet/mm-choosebot.png" width="400" alt="">
+
+After configuring placements and other parameters, the bot can be created by clicking the `Create Bot` button.
+
+<img src="./images/using-bison-wallet/mm-basic-createbot.png" width="400" alt="">
+
+The Market Making Overview screen will appear, displaying all configured bots.
+
+<img src="./images/using-bison-wallet/mm-basic-createbot2.png" width="800" alt="">
+
+To start the bot, click the `Allocate and Run` button. A prompt will appear, confirming whether you have sufficient funds to run the bot with the configured parameters.
+
+<img src="./images/using-bison-wallet/mm-allocate-1.png" width="400" alt="">
+ 
+The bot can then be started by pressing the <img src="./images/using-bison-wallet/mm-start-button.png" width="20" alt=""> button.
+
+The market making screen will update to display the bot's status and key performance statistics. 
+
+<img src="./images/using-bison-wallet/mm-basic-running.png" width="800" alt="">
+
+The bot can be stopped by pressing the `Stop` button, or deleted by pressing the <img src="./images/using-bison-wallet/mm-delete-button.png" width="20" alt=""> button. 
```

Comment:
The delete button you're showing is the view logs button.

Reply:
Yeah I noticed this after I pushed this, will correct - I will be adding the arb+mm and simple arb.